discovery/bigqueryreservation_v1beta1.json

```diff
@@ -994,11 +1039,6 @@
           "description": "If false, any query using this reservation will use idle slots from other reservations within the same admin project. If true, a query using this reservation will execute with the slot capacity specified above at most.",
           "type": "boolean"
         },
-        "maxConcurrency": {
```

Comment:
Is this a breaking change?

Reply:
Yes, it looks like a breaking change to me

Reply:
What's the intended resolution then? Add a changelog note and take the change?

Reply:
At the moment yes. This may trigger a version bump once we are out of preview.